Simplenote.xcodeproj/project.pbxproj

```diff
@@ -2409,7 +2409,6 @@
 				SDKROOT = macosx;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_VERSION = 5.0;
-				VALID_ARCHS = x86_64;
```

Comment:
This appears to be a legacy build setting: if you create a new macOS with Xcode 12.0, it won't be there.

I opted for removing it instead of updating it to include `arm64`.

Reply:
Apparently it defaults to `Standard Archs` so it's all good! thank you!!